convene-web/.overmind.env

```diff
@@ -0,0 +1 @@
+OVERMIND_PROCFILE=Procfile.dev
```

Comment:
My instincts are that overmind is a project-level dependency, not a module-level one; so perhaps we could have a Procfile in the top-level directory that starts maildev, the web server, the asset compilation process, and (when we have them) background jobs?